ondemand/install.sh

```diff
@@ -10,50 +10,88 @@ log_info() {
 log_info "Installing required packages for Ondemand.."
 yum install -y \
     centos-release-scl \
-    https://yum.osc.edu/ondemand/1.7/ondemand-release-web-1.7-1.noarch.rpm
+    https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
 
 yum install -y \
     ondemand \
-    openssl \
-    mod_authnz_pam
+    ondemand-dex \
+    openssl
 
 log_info "Setting up Ondemand"
 mkdir -p /etc/ood/config/clusters.d
 mkdir -p /etc/ood/config/apps/shell
 echo "DEFAULT_SSHHOST=frontend" > /etc/ood/config/apps/shell/env
 
-
-log_info "Configuring Ondemand for PAM based logins.."
-
-# This is to allow ondemand to use pam for local logins in docker:
-chmod 640 /etc/shadow
-chgrp apache /etc/shadow
-
-echo "LoadModule authnz_pam_module modules/mod_authnz_pam.so" > /opt/rh/httpd24/root/etc/httpd/conf.modules.d/55-authnz_pam.conf
-cp /usr/lib64/httpd/modules/mod_authnz_pam.so /opt/rh/httpd24/root/usr/lib64/httpd/modules/
-cp /etc/pam.d/sshd /etc/pam.d/ood-webapp
+log_info "Configuring Ondemand for LDAP based logins.."
```

Comment:
Is this still a valid comment?  Doesn't seem like it actually doing anything

Reply:
nevermind, just noticed the yml with the dex configuration.

Reply:
Sort of?  In reality what we are doing is configuring OnDemand as we setup `ood_portal.yml` which is the core OnDemand config file for Apache and Dex.  The configuration of Dex is to talk to LDAP to the log message is accurate just maybe not specific enough?

Reply:
Updated wording to be more accurate